taf/docker-compose-taf.yml

```diff
@@ -29,7 +29,7 @@ networks:
     driver: bridge
 services:
   app-service-functional-tests:
-    container_name: app-service-functional-tests
+    container_name: app-functional-tests
```

Comment:
Looks like we need to figure out how to add the security settings for app-service-functional-tests

Reply:
if `app-service-functional-test` would be tested in both modes, i.e., with security and no-security modes, then probably we would do the similar way like `asc-x` with base and extending it with -secure mode.

@cherrycl  is TAF testing treating this `app-functional-tests` the same way as `asc-x`?

Reply:
It is only used in TAF, so I don't want to clutter those files. Just need `add-taf-app-services-secure.yml``  & `add-taf-device-services-secure.yml`

Reply:
device services are out of scope for this PR; we'll deal with devices services in different PR

Reply:
Ok.

Reply:
added